File: py2many/add_method_3.go
package main

type Foo struct {
	_value int
}

func __init__(self Foo, value int) {
	self._value = value
}

func __add__(self Foo, other Foo) Foo {
	return Foo{_value: (self._value + other._value)}
}

func __str__(self Foo) string {
	return ("*" * self._value)
}

File: py2many/add_method_3.jl
struct Foo
    _value::Int64
end

function __init__(self::Foo, value::Int64)
    self._value = value
end

function __add__(self::Foo, other::Foo)::Foo
    return Foo(self._value + other._value)
end

function __str__(self::Foo)::String
    return "*"*self._value
end

